foundry.toml

```diff
@@ -1,9 +1,10 @@
 [profile.default]
-src = 'src'
+src = 'contracts'
 out = 'out'
 libs = ['lib']
 solc_version = '0.8.18'
 test = 'tests'
+ignored_warnings_from = ["contracts/Teleporter/tests", "contracts/Teleporter/upgrades/tests"]
```

Comment:
Some of the test files were throwing "Function state mutability can be restricted to pure/view" warnings that failed to compile when the suggested change was applied. Unfortunately forge doesn't support wildcard filters for this option, so we'll have to keep an eye on emitted warnings for any future tests we add.

Reply:
Is there any reason why we couldn't fix the state mutability of those functions?

Reply:
Fixed now 👍 I had my wires crossed while putting the original change together.